pkg/asset/imagebased/configimage/ingressoperatorsigner.go

```diff
@@ -72,3 +90,75 @@ func (a *IngressOperatorCABundle) Generate(ctx context.Context, deps asset.Paren
 func (a *IngressOperatorCABundle) Name() string {
 	return "Certificate (ingress-operator-ca-bundle)"
 }
+
+// selfSignedCertificate creates a self signed certificate.
+func selfSignedCertificate(cfg *tls.CertCfg, key *rsa.PrivateKey) (*x509.Certificate, error) {
```

Comment:
This function is a modified clone of https://github.com/openshift/installer/blob/f90d82ab3bb39ddcd6381e205b4b1a49c1309aa8/pkg/asset/tls/tls.go#L63-L91 just so we the organizational unit? Could we share some of that code instead by making the original more flexible?

Reply:
Unfortunately yes, I tried to find an easier way, i.e. share as much code as possible, but I couldn't reach something better than cloning all those pieces. I'll give it another try though, because it does indeed make sense to not clone all this just for the OU field.

Reply:
Just saw I didn't really answer your question, my thinking was to have this issue fixed and then come back with another PR that would reduce this tech debt by updating `pkg/asset/tls`.

Reply:
Sounds reasonable, just leave a TODO comment indicating that this function is merely a slightly modified clone of <link to original>